qa/setup_packages.py

```diff
@@ -183,7 +206,9 @@ def get_install_string(self, idx, cuda_version=None):
             `cuda_version`: str, optional, default = None
                 Cuda version used for this query
         """
-        return "{name}=={version}".format(name=self.get_name(cuda_version, idx), version=self.get_version(idx, cuda_version))
+        pkg_cmd = "{name}=={version}".format(name=self.get_name(cuda_version, idx), version=self.get_version(idx, cuda_version))
+        deps_cmd = " ".join(self.get_dependencies(cuda_version, idx))
+        return " ".join([pkg_cmd, deps_cmd])
```

Comment:
```suggestion
        return " ".join([pkg_cmd] + self.get_dependencies(cuda_version, idx))
```
would there be any difference with this? I mean, can we make it a single `join` call?

Reply:
Fixed